package.json

```diff
@@ -30,7 +30,8 @@
     "test": "node Makefile.js test",
     "test:cli": "mocha",
     "test:fuzz": "node Makefile.js fuzz",
-    "test:performance": "node Makefile.js perf"
+    "test:performance": "node Makefile.js perf",
+    "trunk": "trunk"
```

Comment:
I'm thinking we'd probably want `"lint": "trunk"` if the intent is to replace the current linting task?

Reply:
yeah - I just need to extract the json file check and then we're in business